scripts/cli

```diff
@@ -186,15 +186,27 @@ cli_download_command() {
   # src/download_command.sh
   root_dir=$(dirname $(dirname $(readlink -f "$0")))
   filename=openapi.yaml
-  declare -A urls=( ["public-api"]="https://edge.staging.cdo.cisco.com/api/platform/public-api/v3/api-docs.yaml" ["object-service"]="https://edge.staging.cdo.cisco.com/api/platform/object-service/v3/api-docs.yaml" )
+  declare -A urls=(
+    ["public-api"]="https://edge.staging.cdo.cisco.com/api/platform/public-api/v3/api-docs.yaml"
+    ["object-service"]="https://edge.staging.cdo.cisco.com/api/platform/object-service/v3/api-docs.yaml"
+    # Currently dependent on the acme tenant cdFMC - need to update
+    ["cdFmc-service"]="https://cdo-acme.app.staging.cdo.cisco.com/api/api-explorer/fmc_oas3.json"
```

Comment:
Instead of pulling it directly (which is risky, because we don't always have the same versions in staging and prod, let's just do the following:
1. Get the latest production cdFMC oas3.json and put it in the local directory.
2. Add a new command to our CLI called `transform-fmc-oas`, which does the mangling you do, and creates `cdfmc-service-openapi.yaml`.
3. Work with @pjayala to make sure that on every cdFMC release, we have a step in our runbook to check out the cdo-public-api-docs repo and update the cdFMC OpenAPI JSON.

Reply:
Addressed 1 and 2, regarding 3 - perhaps we could as part of our release process, run the download command to fetch the latest version from `api-explorer/fmc_oas3.json`?